test/test_files/tinysnb/function/string.test

```diff
@@ -7,15 +7,28 @@
 
 -LOG StrAddOperation
 -STATEMENT MATCH (a:person) RETURN a.fName + "suffix"
----- 8
-Alicesuffix
-Bobsuffix
-Carolsuffix
-Dansuffix
-Elizabethsuffix
-Farooqsuffix
-Gregsuffix
-Hubert Blaine Wolfeschlegelsteinhausenbergerdorffsuffix
+---- error
```

Comment:
I would still suggest we support + overloading for string

Reply:
If we keep string + string, and allow any type -> string implicit cast, then users could add any datatype together